performance_regression_test.py

```diff
@@ -40,6 +40,7 @@ class PerformanceRegressionTest(ClusterTester):  # pylint: disable=too-many-publ
     start_ops = 10_000
     throttle_step = 10_000
     max_ops = 200_000
+    base_index = "stagingperftest"
```

Comment:
you might want to have some degree of control here, where would not staging results would go  ?

Reply:
Thanks, replace this logic to minize affect on other tests. Added new config parameter to setup specific es_index